core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -395,7 +395,12 @@ public MetricKey build() {
           .setDescription("Length of the master rpc queue")
           .setMetricType(MetricType.GAUGE)
           .build();
-
+  public static final MetricKey MASTER_ACTIVE_JOB_SIZE =
```

Comment:
could you make the name more explicit, so that it indicates jobs started by master and not total active jobs? somethng like MASTER_HEARTBEAT_TRIGGERED_ACTIVE_JOB_SIZE?

Reply:
@yuzhu Done, PTAL!